core/src/components/picker-internal/picker-internal.vars.scss

```
@@ -0,0 +1,2 @@
+$picker-fade-background-fallback: var(--background-rgb, var(--ion-background-color-rgb, 255, 255, 255));
+$picker-fade-background: var(--picker-fade-background-rgb, #{$picker-fade-background-fallback});
```

```suggestion
$picker-fade-background: var(--picker-fade-background-rgb, $picker-fade-background-fallback);
```

I do not believe we need to escape the variable here.